style1.css

```diff
@@ -149,7 +149,7 @@
   color: #fff;
   font-family: "Crete Round", sans-serif;
   font-weight: normal;
-  font-size: 32px;
+  font-size: 31px;
   margin-left: 16px;
 }
 
```

Comment:
Hi there, you have done an incredible job 👍  following the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848) for your desktop version of the portfolio project, nevertheless, there are some issues that we would need to address. 
Permit me make the inline comments here, since most of them are not applying to a line or block of code, but to some project requirements that you missed.

- [ ] The project requirement states clearly that you should use Media queries in your CSS file to make your desktop version look like the template. Knowing how to use media queries effectively is very importnant because today most people use different types of devices and with media queries you will be able to give the same user experience to users of different sizes of devices. But I have gone through your entire CSS file and I think I have not seen where you applied media queries. I also think this is because you have created a new CSS file for this desktop version rather than using the same CSS file you used for the mobile version. Please kindly use media queries to set your desktop breakpoint. The value for `min-width` property should be in range of `992px` - `1100px`. See example below.
````css
@media screen and (min-width: 1080px) { // please note that it doesn’t have to be exactly 1080px, any value between 992 and 1100 is ok.
/*
.
.
*/
}
````
![image](https://user-images.githubusercontent.com/62849872/144018001-32fc1274-10eb-43cf-9c90-4b76cffd9324.png)